src/ast/mod.rs

```diff
@@ -7116,6 +7123,63 @@ where
     }
 }
 
+/// Represents a list of options used in various PostgreSQL utility statements such as
+/// `EXPLAIN`, `VACUUM`, and `CLUSTER`.
+///
+/// For example, the `EXPLAIN` statement with options might look like this:
+///
+/// ```sql
+/// EXPLAIN (ANALYZE, VERBOSE true) SELECT * FROM my_table;
+/// ```
+#[derive(Debug, Clone, PartialEq, PartialOrd, Eq, Ord, Hash)]
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+#[cfg_attr(feature = "visitor", derive(Visit, VisitMut))]
+pub struct UtilityOptionList {
+    pub options: Vec<UtilityOption>,
+}
+
+impl Display for UtilityOptionList {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write!(f, "( ")?;
+
+        let mut iter = self.options.iter().peekable();
+        while let Some(option) = iter.next() {
+            write!(f, "{}", option)?;
+            if iter.peek().is_some() {
+                write!(f, ", ")?;
+            }
+        }
+
+        write!(f, " ) ")
+    }
+}
+
+/// Represents a single PostgreSQL utility option.
+///
+/// A utility option is a key-value pair where the key is an identifier (IDENT) and the value
+/// can be one of the following:
+/// - A number with an optional sign (`+` or `-`)
+/// - A non-keyword string
+/// - keyword: `true`, `false`, `on` (`off` is also accept)
+/// - Empty
```

Comment:
I think we can give actual examples to illustrate but also skip describing the syntax in words since that can quickly go out of date or become ambiguous - (inline examples in addition to a link to the docs should be good enough and would remain consistent)

Reply:
Thank you. Add some actual examples.